lib/components/molecules/DateSelector/DateSelector.js

```diff
@@ -0,0 +1,203 @@
+// @flow
+/**
+ *
+ * DateSelector
+ *
+ */
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import classNames from 'classnames';
+import styles from './DateSelector.style';
+import type { Props } from './types';
+import Select from '../../atoms/Select';
+import Label from '../../atoms/Label';
+
+const monthOptions = [
```

Comment:
How will these change if locale changes to lets say French or Spanish

Reply:
I think we can get this dynamically, from browser itself and then we will no longer need to take care of locales

Reply:
implemented